tests/e2e/example-test/02-assert.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 30
+namespaced: true
+commands:
+  - script: |
+      PASSWORD=$(kubectl -n $NAMESPACE get secrets grafana-admin-credentials --template={{.data.GF_SECURITY_ADMIN_PASSWORD}} | base64 -d)
+      echo $PASSWORD
+      kubectl port-forward -n grafana-operator-system service/grafana-service 3000:3000 &
+      FPID=$!
+      sleep 1
+      curl localhost:3000/api/health
+      kill $FPID
```

Comment:
I feel like the "nicer" way of doing assertions in general would be to have the operator put something in the status and assert on that. For this specifically, since it is a health check, I would argue  that the assertion should be on the status of the pod.

Reply:
The health check is just a way for me to know that I can see that the port-forward is working as intended.

For the long run it would be preferable to have statuses inside the dashboards and look if those are correct.
Then we wouldn't have to do any port-forward at all and could just look at the created resources.

But that is out of scope for this PR, but something that we should look in to for the future.